crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -38,9 +38,10 @@ pub enum Error {
 }
 
 /// Rollup data that relayer/conductor need to know.
-#[derive(Default, Serialize, Deserialize, Clone, Debug, Eq, PartialEq)]
```

Comment:
Removed this because `ChainId` has no natural default: chain ID is constructed by hashing some bytes using sha256, and a zero-hash is a valid sha256 hash.